.github/workflows/backup.yaml

```diff
@@ -0,0 +1,33 @@
+name: Backup
+
+on:
+  workflow_dispatch:
+env:
+  IMAGE_NAME: backup
+  WORKING_DIRECTORY: ./
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:      
+
+      - name: Login to Artifactory
+        uses: docker/login-action@v1
+        with:
+          registry: artifacts.developer.gov.bc.ca
+          username: ${{ secrets.ARTIFACTORY_USERNAME }}
+          password: ${{ secrets.ARTIFACTORY_PASSWORD }}
+
+      - name: Checkout Schema Spy Repo
+        id: gitops
+        uses: actions/checkout@v4
+        with:
+          repository: BCDevOps/backup-container.git
+          ref: master                    
```

Comment:
The `ref` should be using a tagged version of the backup-container rather than the `master` branch.